src/screens/EmergencyContacts.js

```diff
@@ -1,29 +1,34 @@
-import React, { useState } from "react";
-import { StyleSheet, Text, View, TouchableOpacity } from "react-native";
-import { TextInput } from "react-native-gesture-handler";
-import { registerNewUser } from "../../firebase/firebase.util";
-import { colors } from "../styles/colors.js";
-import { AppLoading } from "expo";
 import {
-  useFonts,
-  Nunito_600SemiBold,
   Nunito_400Regular,
+  Nunito_600SemiBold,
   Nunito_800ExtraBold,
+  useFonts,
 } from "@expo-google-fonts/nunito";
-import { CoveredByYourGrace_400Regular } from "@expo-google-fonts/covered-by-your-grace";
 import {
-  // Quicksand_300Light,
   Quicksand_400Regular,
   Quicksand_500Medium,
   Quicksand_600SemiBold,
   Quicksand_700Bold,
 } from "@expo-google-fonts/quicksand";
+import React, { useState } from "react";
+import { StyleSheet, Text, TouchableOpacity, View } from "react-native";
+
+import { AppLoading } from "expo";
+import { CoveredByYourGrace_400Regular } from "@expo-google-fonts/covered-by-your-grace";
+import { TextInput } from "react-native-gesture-handler";
+import { colors } from "../styles/colors.js";
+import { registerNewUser } from "../../firebase/firebase.util";
 
 // TODO: remove extraneous comments
 const EmergencyContacts = ({ navigation }) => {
+  // [Ian]: I would recommend updating to React Navigation 5.x.x so you can use Hooks and
```

Comment:
To add to this, I recommend Hooks so that you don't have to pass the prop around constantly. For example, instead of passing in the navigation prop, you would use 
```suggestion
  const navigation = useNavigation();
  ...
  navigation.navigate(someRoute)
```
and then for your `getParam` you could use the route hook:
```suggestion
  const route = useRoute();
  const { object, otherObject } = route.params;